tests/components/mqtt/test_device_trigger.py

```diff
@@ -513,10 +511,10 @@ async def test_no_resubscribe_same_topic(hass, device_reg, mqtt_mock):
         },
     )
 
-    call_count = mock_mqtt.async_subscribe.call_count
+    call_count = mqtt_mock.async_subscribe.call_count
     async_fire_mqtt_message(hass, "homeassistant/device_automation/bla1/config", data1)
     await hass.async_block_till_done()
-    assert mock_mqtt.async_subscribe.call_count == call_count
+    assert mqtt_mock.async_subscribe.call_count == call_count
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5737505)